### ISSUE TITLE
Mismatched useCapture flag not reported

### ISSUE THREAD

**Issue by erikjalevik** [foad/eslint-plugin-listeners#4]
I just installed your plugin due to finding a devious bug where I had an `addEventListener` call with the `useCapture` argument set to true, and a `removeEventListener` call without the flag. This doesn't actually remove it, see [MDN](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/removeEventListener).

In other words, the following code is not flagged as a problem right now, and it probably should be:

```
private startListening() {
  const useCapture = true;
  document.addEventListener("mousemove", this.onMouseMove, useCapture);
}

private stopListening() {
  document.removeEventListener("mousemove", this.onMouseMove);
}
```

**Comment by foad** [comment]
Thanks for raising, should be fixed in https://github.com/foad/eslint-plugin-listeners/releases/tag/v1.4.0

N.B. that if the useCapture parameter is passed in as a variable like in your example, there's no way to verify that the values match since ESLint can't evaluate expressions, so it can only check that _something_ was set for that useCapture argument for both the add/remove event listeners

![Screenshot 2023-10-20 at 13 27 51](https://github.com/foad/eslint-plugin-listeners/assets/8802986/3c75fa3d-a3b5-4105-8b40-84c9f6691c13)
